config/job_conf.xml

```diff
@@ -223,14 +223,14 @@
             <param id="max_pod_retrials">3</param>
             <param id="docker_enabled">true</param>
         </destination>
-        <destination id="container-isajson-validator" runner="k8s">
+        <destination id="container-mw2isa" runner="k8s">
```

Comment:
Question @proccaserra : why are you overwritting container-isajson-validator with container-mwisa?